packages/renderer/src/lib/preferences/PreferencesKubernetesConnectionRendering.svelte

```diff
@@ -193,7 +193,7 @@ async function stopReceivingLogs(provider: ProviderInfo): Promise<void> {
     </div>
 
     {#if lifecycleError}
-      <div class="text-red-600">
+      <div class="text-red-500">
```

Comment:
not for this PR (just thinking out loud) but probably we should introduce a <ErrorMessage> component that will set the color and allowing us to easily use a common color

Reply:
https://github.com/containers/podman-desktop/issues/1365